llvm/lib/IR/VFABIDemangler.cpp

```diff
@@ -42,6 +42,7 @@ static ParseRet tryParseISA(StringRef &MangledName, VFISAKind &ISA) {
     ISA = StringSwitch<VFISAKind>(MangledName.take_front(1))
               .Case("n", VFISAKind::AdvancedSIMD)
               .Case("s", VFISAKind::SVE)
+              .Case("v", VFISAKind::RVV)
```

Comment:
Is this letter choice internal to LLVM or is it agreement with SLEEF? I'm surprised we don't use "r" for RISC-V.

Reply:
Now, there is no VFABIDemagler support for RISC-V. Letter `r` also can be used.

Reply:
Changed to 'r' letter